projects/netgrif-components/package.json

```diff
@@ -30,35 +30,36 @@
     ],
     "peerDependencies": {
         "@netgrif/components-core": "6.1.0-rc.1",
-        "@angular-material-components/datetime-picker": "^2.0.2",
-        "@angular-material-components/moment-adapter": "^2.0.1",
-        "@angular/animations": "^9.1.12",
-        "@angular/cdk": "^9.2.4",
-        "@angular/common": "^9.1.12",
-        "@angular/compiler": "^9.1.12",
-        "@angular/core": "^9.1.12",
-        "@angular/flex-layout": "^9.0.0-beta.31",
-        "@angular/forms": "^9.1.12",
-        "@angular/material": "^9.2.4",
-        "@angular/material-moment-adapter": "^9.2.4",
-        "@angular/platform-browser": "^9.1.12",
-        "@angular/platform-browser-dynamic": "^9.1.12",
-        "@angular/router": "^9.1.12",
-        "@covalent/core": "^3.0.1",
-        "@covalent/highlight": "^3.0.1",
-        "@covalent/markdown": "^3.0.1",
-        "@covalent/text-editor": "^3.0.1",
-        "@ngx-translate/core": "^12.1.2",
-        "@ngx-translate/http-loader": "^5.0.0",
-        "@schematics/angular": "^9.1.11",
-        "angular-resize-event": "^2.0.1",
-        "angular2-hotkeys": "^2.2.0",
-        "hammerjs": "^2.0.8",
-        "json-schema-to-typescript": "^8.1.0",
-        "jszip": "^3.2.2",
+        "@angular-material-components/datetime-picker": "~7.0.1",
+        "@angular-material-components/moment-adapter": "~7.0.0",
+        "@angular/animations": "~13.3.1",
+        "@angular/cdk": "~13.3.1",
+        "@angular/common": "~13.3.1",
+        "@angular/compiler": "~13.3.1",
+        "@angular/core": "~13.3.1",
+        "@angular/flex-layout": "~13.0.0-beta.38",
+        "@angular/forms": "~13.3.1",
+        "@angular/material": "~13.3.1",
+        "@angular/material-moment-adapter": "~13.3.1",
+        "@angular/platform-browser": "~13.3.1",
+        "@angular/platform-browser-dynamic": "~13.3.1",
+        "@angular/router": "~13.3.1",
+        "@covalent/core": "^4.0.1",
```

Comment:
covalent with ^ other dependencies with ~ why?

Reply:
it was fixed by @renczesstefan